runtime/vm/BytecodeInterpreter.cpp

```diff
@@ -38,7 +38,7 @@
 /* USE_COMPUTED_GOTO on Linux amd64 has a performance improvement of 3% */
 #if (defined(WIN32) && defined(__GNUC__))
 #define USE_COMPUTED_GOTO
-#elif (defined(LINUX) && defined(J9HAMMER))
+#elif (defined(LINUX) && (defined(J9HAMMER) || defined(S390)))
```

Comment:
What happens if you try to compile the BytecodeInterpreter on Linux 390 with xlC? Is this possible with our current makefiles?

Reply:
Not sure this has ever been tried.